Lib/test/test_idle/test_config.py

```diff
@@ -185,12 +185,9 @@ def setUpClass(cls):
         cls.config_string = {}
 
         conf = config.IdleConf(_utest=True)
-        if __name__ != '__main__':
-            idle_dir = os.path.dirname(__file__)
-        else:
-            idle_dir = os.path.abspath(sys.path[0])
+        idle_dir = os.path.abspath(os.path.dirname(idlelib.__file__))
```

Comment:
I believe all these test file changes make them independent of test file location.  If so, they can be done now without moving the files.  It would be easy to find out.  Some of these look like improvements in any case.  I would add a note to idle_test.README.txt that tests should not depend on being in idle_test.